### PR TITLE
add-search

### PR DESCRIPTION
**Pull request by N-Patarov**

### CODE REVIEW

File: lib/bgsite_official/home/websites.ex [hunk 1]
```diff
@@ -25,4 +26,10 @@ defmodule BgsiteOfficial.Home.Websites do
   #  |> cast(attrs, [])
   #  |> cast_attachments(attrs, [:banner])
   # end
+  def search(query, search_term)do
+    wildcard_search = "%#{search_term}"
+
+    from websites in query,
+    where: ilike(websites.title,^wildcard_search)
```
Comment (rmilushev) [inline lib/bgsite_official/home/websites.ex:33]: Let's search by `description` field, it will be more cool :)

**Review by rmilushev — CHANGES_REQUESTED** [review]

File: lib/bgsite_official_web/router.ex [hunk 2]
```diff
@@ -19,8 +19,7 @@ defmodule BgsiteOfficialWeb.Router do
 
   scope "/", BgsiteOfficialWeb do
     pipe_through :browser
-
-    live "/", PageLive, :index
+    get "/", WebsitesControlller, :index
```
Comment (rmilushev) [inline lib/bgsite_official_web/router.ex:22]: My bad, it has to be: `WebsitesController`, not triple `l` :)

File: lib/bgsite_official_web/templates/layout/root.html.leex [hunk 3]
```diff
@@ -31,13 +32,24 @@
...
+  <%= form_for @conn, Routes.websites_path(@conn, :index),[
+    method: :get,
+    class: "pull-right"], fn f -> %>
+    <class="input-group-text border-0" id="search-addon">
+    <%= search_input f, :query%>
+    </class>
+    <%= submit "Search"%>
+  <%end%>
+
+  <% @inner_content %>
```
Comment (ykostov) [inline lib/bgsite_official_web/templates/layout/root.html.leex:52]: We don't need 2nd inner_content, plz delete it

**Review by ykostov — CHANGES_REQUESTED** [review]

File: lib/bgsite_official_web/templates/layout/root.html.leex [hunk 4]
```diff
@@ -31,13 +32,24 @@
...
-  <input type="search" class="form-control rounded" placeholder="Search" aria-label="Search"
+ <!--<div class="input-group rounded">
+  <input type="search" class="form-control rounded"  placeholder="Search" aria-label="Search"
     aria-describedby="search-addon" />
   <span class="input-group-text border-0" id="search-addon">
     <i class="fas fa-search"></i>
-  </span>
+  </span>-->
 </div>
+  <class="search">
```
Comment (rmilushev) [inline lib/bgsite_official_web/templates/layout/root.html.leex:42]: Is this a new syntax, strange, imo it has to be:
```<div class=...>```

File: lib/bgsite_official_web/templates/layout/root.html.leex [hunk 5]
```diff
@@ -31,13 +32,24 @@
...
+  <%= form_for @conn, Routes.websites_path(@conn, :index),[
+    method: :get,
+    class: "pull-right"], fn f -> %>
+    <class="input-group-text border-0" id="search-addon">
+    <%= search_input f, :query%>
+    </class>
```
Comment (rmilushev) [inline lib/bgsite_official_web/templates/layout/root.html.leex:48]: Probably ```</div>```?

**Review by rmilushev — CHANGES_REQUESTED** [review]

File: lib/bgsite_official_web/templates/layout/root.html.leex (hunk 4 above)
Comment (ykostov) [reply lib/bgsite_official_web/templates/layout/root.html.leex:42]: Agree

File: lib/bgsite_official_web/templates/layout/root.html.leex (hunk 3 above)
Comment (N-Patarov) [reply lib/bgsite_official_web/templates/layout/root.html.leex:52]: nqma 2ri

**Review by ykostov — APPROVED** [review]
LGTM

**Pull request merged**